fineract-provider/build.gradle

```diff
@@ -57,7 +57,7 @@ apply plugin: 'eclipse'
 apply plugin: 'idea'
 apply plugin: 'com.bmuschko.tomcat'
 apply plugin: 'project-report'
-apply plugin: 'java'
+apply plugin: 'java-library'
```

Comment:
@percyashu could you reply with a note for future code archeologists here why this change was needed?  As it seems to still work anyway, I'm inclined to merge this even without fully understanding what exactly this changes, if anything.